packages/frontend/components/Shared/ConfirmTransactionModal.tsx

```diff
@@ -131,7 +131,12 @@ export function ConfirmTransactionModal({
           <></>
         )}
 
-        {steps && steps.length > 0 && <RouteBox steps={steps} />}
+        {steps && steps.length > 0 && (
+          <RouteBox
+            steps={steps}
+            isCrossChainTransaction={collateral.chainId !== debt.chainId}
```

Comment:
This is not a precise way to check if a tx is a cross-chain one because not always we have both collateral and debt (We can only have a deposit action and do this from another chain which makes it a cross-chain tx).

You can check if a tx is cross-chain by comparing `chainId` of the START step and the END step.
```
const start = steps.find(RoutingStep.START)
const end  = steps.find(RoutingStep.END)
const isCrossChain = start.chainId !== end.chainId
```